ArduPlane/system.cpp

```diff
@@ -162,7 +162,7 @@ void Plane::init_ardupilot()
     // setup frsky, and pass a number of parameters to the library
     frsky_telemetry.init(serial_manager, FIRMWARE_STRING,
                          MAV_TYPE_FIXED_WING,
-                         &g.fs_batt_voltage, &g.fs_batt_mah);
+                         &g.fs_batt_voltage, &g.fs_batt_mah, nullptr);
```

Comment:
If it is always the last parameters that can be nullptr, then declare the init function with nullptr for those, then you don't need to pass nullptr arguments.